<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WireGuardInterfaceProp</class>
 <widget class="QWidget" name="WireGuardProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WireGuard Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Interface</string>
     </property>
     <layout class="QFormLayout" name="form1Layout">
      <item row="0" column="0">
       <widget class="QLabel" name="privateKeyLabel">
        <property name="text">
         <string>Private key:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PasswordField" name="privateKeyLineEdit">
        <property name="toolTip">
         <string>Required.
A base64 private key generated by wg genkey.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="listenPortLabel">
        <property name="text">
         <string>Listen port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="listenPortLineEdit">
        <property name="toolTip">
         <string>Optional.
Listen port number. Chosen randomly if left as 'Automatic'.</string>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="fwmarkLabel">
        <property name="text">
         <string>fwmark:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="fwmarkLineEdit">
        <property name="toolTip">
         <string>Optional.
An fwmark for outgoing packets. If set to 0 or 'off', this
option is disabled. May be specified in hexadecimal by
prepending '0x'.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mtuLabel">
        <property name="text">
         <string>MTU:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mtuLineEdit">
        <property name="toolTip">
         <string>Optional.
If not specified, the MTU is automatically determined
from the endpoint addresses or the system default route,
which is usually a sane choice. However, to manually
specify an MTU and to override this automatic discovery,
this value may be specified explicitly.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="peerRouteLabel">
        <property name="text">
         <string>Autoroute peers:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="peerRouteCheckBox">
        <property name="toolTip">
         <string>Whether to automatically add routes for the AllowedIPs ranges
of the peers.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnPeers">
       <property name="text">
        <string>Peersâ€¦</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>


   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordField</class>
   <extends>QLineEdit</extends>
   <header location="global">passwordfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
